model-optimizer/unit_tests/mo/utils/versions_checker_test.py

```diff
@@ -154,3 +165,23 @@ def test_version_check_greater(self):
         for name, key, required_version, installed_version in modules_versions_list:
             version_check(name, installed_version, required_version, key, not_satisfied_versions)
         self.assertEqual(not_satisfied_versions, ref_list)
+
+    def test_version_check_compatible(self):
+        modules_versions_list = [('module_1', '~=', '1.2.3', '1.2.3'),
```

Comment:
Add unit tests provided here https://www.python.org/dev/peps/pep-0440/#compatible-release

I think that the current implementation will fail the following test: `~= 2.2.post3`

Reply:
Yes, it fell. Implementation was updated and more unit tests was provided.